test/local_integration_test.py

```diff
@@ -121,9 +121,8 @@ def _test_other_endpoints(self):
     def _test_manifest(self):
         manifest_filter = json.dumps({'project': {'is': [self.test_name]}})
         for format_, validator in [
-            (None, self.check_manifest),
-            ('tsv', self.check_manifest),
-            ('bdbag', self.check_bdbag)
+            ('tsv', self.check_manifest),  # TODO: Validate `full` format when
```

Comment:
Right. Please create ticket so we can track this TODO and reference that ticket here.